CC/Sounder/files/bs-measurements-2.5g.json

```diff
@@ -1,14 +1,14 @@
 {
 	"BaseStations" : {
-		 "cells" : 1,
-		 "sdr_id" : ["files/measurements-iris-serials.txt"],
+		 "cells" : 4,
+		 "sdr_id" : ["files/bs145.txt", "files/bs166.txt", "files/bs030.txt", "files/bs179.txt"],
 		 "hub_id" : "files/measurements-hub-serials.txt",
 		 "frequency" : 2.5e9,
 		 "channel" : "AB",
-		 "rxgainA" : 20,
-		 "txgainA" : 85,
-		 "rxgainB" : 20,
-		 "txgainB" : 85,
+		 "rxgainA" : 70,
+		 "txgainA" : 70,
+		 "rxgainB" : 70,
+		 "txgainB" : 70,
 		 "rate" : 5e6,
 		 "frame_schedule" : [
 		     "GGGGGGGPGGGGGGG"
```

Comment:
Maybe I'm missing something but there is no beacon scheduled on the downlink for sync?

Reply:
done!